.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,12 +14,13 @@ instructions.
 1. Open page A
 2. Do thing B
 3. Expect C to happen
-4. If D happend instead - check failed.
+4. If D happened instead - check failed.
 
 **Merge checklist:**
 
-- [ ] Check that the versions of the requirements in the `platform-master.in` file match edx-platform.
-- [ ] New requirements are in the right place (`base.in` if only used in enterprise; in the correct `platform-****.in` files if they're hosted in edx-platform)
+- [ ] New requirements are in the right place
+    - `base.in` if needed in production, but edx-platform doesn't install it.
```

Comment:
The `but edx-platform doesn't install it.` bit here... I thought we decided that requirements should be in base.in if they are used at all in edx-enterprise (regardless if they are also installed in edx-platform)

Reply:
This has been confusing because I started over with a simpler approach.  In this PR, `test-master.in` is the set of packages that are shared with edx-platform.  `base.in` is for packages only used by edx-enterprise.